libraries/core-react/src/components/Card/CardMedia.tsx

```diff
@@ -11,21 +11,18 @@ export type CardMediaProps = {
 } & HTMLAttributes<HTMLDivElement>
 
 const StyledCardMedia = styled.div<CardMediaProps>`
-  display: flex;
-  width: 100%;
+  display: grid;
```

Comment:
nitpicking, but I think `place-items` still works with `display:flex`?

Reply:
`width:auto` was added and, `display:flex` is back. It looks that component doesn't need `place-items`